CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Braintree iOS Drop-in SDK - Release Notes
 
+## unreleased
+* Update Vault Manager inline documentation 
```

Comment:
I'm not sure we need a CHANGELOG entry for this - does anyone feel strongly one way or the other?

Reply:
@Epreuve and I debated that but figured we could always remove it if anyone felt strongly one was or the other. I personally don't feel strongly about it either way.

Reply:
On second thought - I think it makes sense to keep it if we want to draw merchant attention to the fact that the behavior of this method is different than previously documented 👍